Common/Tests/Utilities.UI/UI/AutomationDialog.cs

```diff
@@ -28,6 +30,7 @@ public AutomationDialog(VisualStudioApp app, AutomationElement element)
             : base(element) {
             App = app;
             DefaultTimeout = TimeSpan.FromSeconds(10.0);
+            _isTaskDialog = element.Current.ClassName == "#32770";
```

Comment:
This looks like an atom. Is it stable?

Reply:
Um, yeah, it's stable, but apparently it's going to be the same for other dialog boxes as well (we were relying on a GetDlgCtrl call succeeding to identify dialog boxes before, but that wasn't working with TaskDialog). But now I've refactored the code that relied on this flag, I can probably just make it more resilient and remove this.